lib/dug/runner.rb

```diff
@@ -33,6 +33,7 @@ def unprocessed_notifications(use_cache: true)
         @unprocessed_notifications = nil
       end
       unprocessed_label = servicer.labels(use_cache: use_cache)[Dug.configuration.unprocessed_label_name]
+      raise "Create the label '#{Dug.configuration.unprocessed_label_name}' on gmail" unless unprocessed_label
```

Comment:
It's auth'd via OAuth, right?  Can it just create the label for you?  Or, maybe have that configurable in the yaml...